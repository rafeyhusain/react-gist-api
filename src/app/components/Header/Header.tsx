import React from 'react';

function Header() {
  return (
    <div className="text-2xl pl-2 leading-relaxed text-left bg-blue-400">
      Gist Explorer
    </div>
  );
}

export default Header;