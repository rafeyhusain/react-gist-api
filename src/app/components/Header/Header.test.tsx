import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer'
import Header from './Header';

describe("Header component tests", () => {
  test("renders without crash", ()=> {
    const div = document.createElement("div");
  
    ReactDOM.render(<Header/>, div);
    ReactDOM.unmountComponentAtNode(div)
  });

  
  test("Snapshot test for Header", ()=> {
    const component = renderer.create(<Header />).toJSON()

    expect(component).toMatchSnapshot()
  });
})
