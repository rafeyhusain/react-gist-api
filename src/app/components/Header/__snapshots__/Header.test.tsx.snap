// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component tests Snapshot test for Header 1`] = `
<div
  className="text-2xl pl-2 leading-relaxed text-left bg-blue-400"
>
  Gist Explorer
</div>
`;
